devinfo/src/mountinfo/io_utils.rs

```diff
@@ -6,10 +6,7 @@ const DEFAULT_RETRY_COUNT: u32 = 2;
 /// Issue std::fs::read on a file twice and compare the two byte sequences
 /// to see if the reads were consistent. Retry if the reads resulted in different byte
 /// sequences.
-pub(crate) fn consistent_read<P: AsRef<Path>>(
-    path: P,
-    retry_count: Option<u32>,
-) -> Result<Vec<u8>> {
+pub fn consistent_read<P: AsRef<Path>>(path: P, retry_count: Option<u32>) -> Result<Vec<u8>> {
```

Comment:
what is the reason to make this accessible directly?

Reply:
No reason. My primary issue was that the 'error' module was private. No real reason for this to be public, and I see no other use-case.

Reply:
Made it private again.